rmarkdown.Rmd

```diff
@@ -157,7 +248,31 @@ model <- lm(Sepal.Length ~ Sepal.Width, data = iris)
 coef(model)
 ```
 
-You can use inline R code too, to insert numbers directly into the document.
+
+#### Exercise 3: adding plots and tables to your results section
+
+Time: 10 min
+
+You now need to create a chunck of code in the **Results section**. If you have any plot from your own research that you want to create feel free to do it so. Otherwise you can just use the iris data:
+
+1. Plot Sepal.length versus Sepal.Width. Include a caption and choose 6 X 10 dimensions.  
+2. Print a table with the mean and sd of each column of the iris dataset. Remember to output just the table and not the code! Use the following code:
```

Comment:
Could have for this task "if you've used dplyr before, try to do this section". Otherwise, we get them to do something even simpler.

Reply:
The goal here is to focus entirely on only one new thing at a time. In this case, only RMarkdown, not on dplyr.